### DISCUSSION TITLE
Hide navigation bar in a private-view

### DISCUSSION THREAD

**Discussion by daawaan4x** [directus/directus#13031] [Archive]
Hello, I'm currently building a small app that only has one dashboard and one collection. Due to the additional list of items present in the navigation bar in insights and content module, I wanted to simplify the interface and I was thinking if I can just go with making a custom module instead to remove the navigation bar. I tried making a custom module with the private-view and it seems like the navigation bar and sidebar are always there by default. Are there any ways to do this?

**Comment by rijkvanzanten** [comment]
There's currently no way to tell the private view to render without the sidebars
